maven-release-manager/src/main/java/org/apache/maven/shared/release/phase/AbstractScmCommitPhase.java

```diff
@@ -288,11 +303,25 @@ protected static List<File> createPomFiles(ReleaseDescriptor releaseDescriptor,
      * @param reactorProjects   a {@link java.util.List} object
      * @return a {@link java.util.List} object
      */
-    protected static List<File> createPomFiles(
-            ReleaseDescriptor releaseDescriptor, List<MavenProject> reactorProjects) {
+    protected List<File> createPomFiles(ReleaseDescriptor releaseDescriptor, List<MavenProject> reactorProjects) {
+
         List<File> pomFiles = new ArrayList<>();
         for (MavenProject project : reactorProjects) {
-            pomFiles.addAll(createPomFiles(releaseDescriptor, project));
+
+            final String path = project.getFile().getPath();
```

Comment:
Might want to use a lambda and anyMatch here. Up to you.

Reply:
I'll use `anyMatch`